cookiecutter.yaml

```diff
@@ -18,21 +28,21 @@ about:
         - url: https://github.com/ICESAT-2HackWeek/ICESat2_hackweek_tutorials
           title: 2019 Hackweek
           new_window: true
-apply:
-  url: '#'
-  title: Application coming soon
-hackweek_mission: https://escience.washington.edu/using-data-science/hackweeks
+# apply:
+#   url: '#'
+#   title: Application coming soon
+# hackweek_mission: https://escience.washington.edu/using-data-science/hackweeks
```

Comment:
Should we keep the button in there for advertisement purposes?

Reply:
I didn't want to leave it there and end up with folks trying to apply to the OT getting to the website and seeing the ambiguous "application coming soon" text... So I'd say keep it and add "Participant" or leave it out. Perhaps we can instead put it as an "application coming soon" button in the info for participants section?

Reply:
Agree with Jessica and noting that my latest changes include an option for participants to signal their interest via our form.